lib/agoneum/web/router.ex

```diff
@@ -25,17 +19,24 @@ defmodule Agoneum.Web.Router do
     plug Guardian.Plug.EnsureResource
   end
 
+  scope "/", Agoneum.Web do
+    pipe_through :browser # Use the default browser stack
+
+    resources "/registrations", RegistrationController, only: [:create, :new]
+
+    get "/", PageController, :index
+  end
+
   scope "/api", Agoneum.Web do
     pipe_through :api
 
     scope "/v1" do
-      post "/registrations", RegistrationController, :create
+      post "/registrations", RegistrationController, :create, as: :api_registration
```

Comment:
Same comment as [this](https://github.com/ElixirCLE/agoneum/pull/14/files#r127532944)